### ISSUE TITLE
Assignment bug in DetConBLoss and CE

### ISSUE THREAD

**Issue by pUmpKin-Co** [isaaccorley/detcon-pytorch#1]
Thanks for your awesome job.
But, in DetConBLoss line(133-134) This parameters passed to CE by (logits, labels, weight). But CE definition is(labels, logits, weight).

**Comment by isaaccorley** [comment]
Thanks for catching this. I'll submit a PR shortly.

**Comment by isaaccorley** [comment]
Fixed in #2
